org.eclipse.xtext.xbase.ui.tests/src/org/eclipse/xtext/xbase/ui/tests/contentassist/Bug471434Test.xtend

```diff
@@ -24,12 +23,8 @@ import org.eclipse.xtext.testing.IInjectorProvider
  * @author Lorenzo Bettini - Initial contribution and API
  */
 @RunWith(typeof(XtextRunner))
-@InjectWith(typeof(ContentAssistFragmentTestLangUiInjectorProviderAdapter))
+@InjectWith(typeof(ContentAssistFragmentTestLangUiInjectorProvider))
```

Comment:
Removed with intention?

Reply:
the adapter was an adapter for old injector to new injector provider. since new workflow no longer needed

Reply:
OK, thought so